docs/source/extensions/strategy.rst

```diff
@@ -16,20 +16,36 @@ The Strategy in PyTorch Lightning handles the following responsibilities:
 * Handles/owns optimizers and schedulers.
```

Comment:
1. nit: L13 and L14 could be merged? 
2. for 16, Strategy also handles move model/data to device, model wrapping, forward, backwards, and optimizer_steps, will only mention `handles/owns optimizer/scheduler`  a bit confusing for users who doesn't know the lightning code base?

Reply:
Going to leave it like this for now. Yes, we can list many more things but these are the core responsibilities and for now we can leave the intro brief.

> a bit confusing for users who doesn't know the lightning code base?

The strategy and accelerators API is more for advanced users and we assume they are not afraid to look at the internals.